tutorials/npm-Enterprise-on-a-closed-network.md

```diff
@@ -0,0 +1,62 @@
+# Npm Enterprise on closed network
+
+You can use npm Enterprise on the private/closed network. Here are the two approaches:
+
+1. Install npmE in open network and move VM’s to closed network
+2. Air-Gap installation
+
+### Install npmE in open network and move VM’s to closed network
+
+To achieve this create two VM’s let’s say npmBlank and npmSource on open-network and install npm Enterprise on both these VM’s.
```

Comment:
Please change to: 
To achieve this, create two VM's (`npmePrimary` and `npmeSource` for example) and install npm Enterprise on both using the instructions in the [Up and Running](https://npme.npmjs.com/docs/up-and-running/platforms/other.html) section.

In this configuration, publication of internal packages and all installs will happen against `npmePrimary` with `npmeSource` serving as the means for importing new public packages and new versions for existing public packages from the registry on a cadence you control.